src/styles/_catalog.scss

```diff
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $width-card;
+  gap: 48px;
   margin: 0 auto;
```

Comment:
Do you need a margin here on the BEM block?

Reply:
fixed